server/chat-commands/admin.ts

```diff
@@ -235,6 +235,44 @@ export const commands: ChatCommands = {
 	sendhtmlpagehelp: [
 		`/sendhtmlpage: [target], [page id], [html] - sends the [target] a HTML room with the HTML [content] and the [pageid]. Requires: * # &`,
 	],
+
+	highlighthtmlpage(target, room, user) {
+		target = target.trim();
+		if (!room) return this.requiresRoom();
+		if (!this.can('addhtml', null, room)) return false;
+		let [userid, pageid, highlight] = Utils.splitFirst(target, ',', 2);
+		pageid = `${user.id}-${toID(pageid)}`;
+		if (!userid || !pageid || !target) return this.parse(`/help highlighthtmlpage`);
+		const targetUser = Users.get(userid);
+		if (!targetUser || !targetUser.connected) {
+			throw new Chat.ErrorMessage(`User ${this.targetUsername} is not currently online.`);
+		}
+		if (targetUser.locked && !this.user.can('lock')) {
+			throw new Chat.ErrorMessage("This user is currently locked, so you cannot send them highlights.");
+		}
+
+		const buf = `|tempnotify|bot-${pageid}|A bot has notified you!|${highlight ? highlight : ''}`;
```

Comment:
```suggestion
		const buf = `|tempnotify|bot-${pageid}|A bot has notified you!|${highlight ? highlight : ''}`;
```

I think "A bot has notified you" is a pretty ugly notification message. Why not let the bot set it? Maybe add `[from ${botName}]` or something before or after it to be clearer.